mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -20,8 +27,23 @@ AUI.add(
 					initializer: function(config) {
 						var instance = this;
 
+						new A.CharCounter(
+							{
+								counter: '.counter',
+								input: '.input-item',
+								maxLength: 35
+							});
+
 						var todoList = instance.one('.task-container ul');
 
+						var finishedTask = A.one('.finished-task');
```

Comment:
Instead of ```instance.one()```, use ```instance.byId()```, ```A.one()``` or ```node.one()```, as previously requested in https://github.com/markocikos/liferay-internship/pull/133#discussion_r118944623 and https://github.com/markocikos/liferay-internship/pull/138#discussion_r121016880

Use ```A.one()``` only if there is no other way, for example is some component adds ```<div>``` in root HTML. Typically you want to query either by ID (```instance.byId()```) or by class relative to container (```node.one()```).